.github/workflows/ci.yml

```diff
@@ -11,14 +11,27 @@ on:
     branches:
       - "**"
 jobs:
+  check-lint:
+    name: Lint
+    timeout-minutes: 5
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup flutter
+        uses: subosito/flutter-action@v2
+        with:
+          cache: true
+      - name: Lint
+        run: flutter format --output=none --set-exit-if-changed packages/flutter
```

Comment:
I think we have to check the format for the dart package, too. This should only check packages/flutter.
I see 3 options:
1. Copy the job (check-lint-dart with dart-lang/setup-dart@v1)
2. use `flutter format --output=none --set-exit-if-changed packages` to format both packages
3. Install flutter and dart in one job and run both format commands

Reply:
I've added a separate job, somehow feels more "correct" to let dart to its own analysis.